src/shared/components/DefaultTooltip.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import Tooltip from 'rc-tooltip';
+import 'rc-tooltip/assets/bootstrap_white.css';
+
+export default class DefaultTooltip extends React.Component<Tooltip.Props, {}> {
+    render() {
+        const defaultProps = {
+            mouseEnterDelay:0.5,
+            mouseLeaveDelay:0.05,
+            arrowContent:(<div className="rc-tooltip-arrow-inner"/>)
+        };
+        return (
+            <Tooltip {...Object.assign({}, this.props, defaultProps)}>
```

Comment:
weird, i thought i commented on this but we should consider reversing this so that a developer can override the defaults if necessary.